src/docfx/lib/JsonUtility.cs

```diff
@@ -162,6 +168,26 @@ public static JObject Merge(JObject first, IEnumerable<JObject> objs)
             return result;
         }
 
+        /// <summary>
+        /// Generate Json schema from object type
+        /// </summary>
+        public static (List<Error>, JSchema) GetJsonSchemaFromType(Type type, string license = null)
+        {
+            var errors = new List<Error>();
+            var licenseRegistrationError = RegisterNewtonJsonSchemaLicense(license);
```

Comment:
Can we call register json schema only once?

Reply:
You are right, I will make register method public for the consumer to invoke just once.

---
In reply to: [208164623](https://github.com/dotnet/docfx/pull/3164#discussion_r208164623) [](ancestors = 208164623)